3.6/views-and-templates/main.md

```diff
@@ -38,9 +38,9 @@ $f3->run();
 
 The only issue with using PHP as a template engine, due to the embedded PHP code in these files, is the conscious effort needed to stick to the guidelines on separation of concerns and resist the temptation of mixing business logic with your user interface.
 
-## A Quick Look at the F3 Template Language
+## Templates: for using the F3 Template Language
 
-As an alternative to PHP, you can use F3's own template engine. The above HTML fragment can be rewritten as:
+As an alternative to PHP (and prevent spaghetti coding), you can use F3's own template engine via the `Template`class. The above HTML fragment can be rewritten as:

```

Comment:
`View` doesn't produce the above mentioned spaghetti code which is created by mixing business with template logic (when following MVC or another suited pattern).